src/DateTime.cc

```diff
@@ -276,7 +306,7 @@ void dafBase::DateTime::setNsecsFromJd(double jd, Timescale scale) {
 /**
  * @brief a function to convert epoch to internal nsecs
  * @param[in] epoch The Julian epoch
- * @param[in] scale The time scale (TAI, or UTC)
+ * @param[in] scale The time scale (TAI, TT or UTC)
```

Comment:
No. This is an existing API that did not need updating as part of this ticket. Also no default follows these principles: "explicit is better than implicit", and "when in doubt, refuse to guess".